lib/manifold.ex

```diff
@@ -59,9 +63,11 @@ defmodule Manifold do
   def send(pids, message, options) when is_list(pids) do
     case options[:send_mode] do
       :offload ->
-        Sender.send(current_sender(), current_partitioner(), pids, message)
+        Sender.send(current_sender(), current_partitioner(), pids, message, options[:pack_mode])
```

Comment:
should we update the spec to reflect the new options?

Reply:
Which spec are you referring to, I updated the `Sender.send/5` function spec here https://github.com/discord/manifold/pull/13/files#diff-170177669413ddc5ed9b1374c8eb5b19a59bfea8fe571ce27cb332a3b9159860R21

Reply:
The one on L59, it should be something like 
```elixir
@type option :: {:pack_mode, pack_mode()} | {:send_mode, :offload}
@spec send([pid | nil] | pid | nil, term, [option]) :: :ok
```
as it supposed to match on a list with any number of pids, any() and a keyword list of options?
maybe I don't know how to write elixir specs anymore, that's a strong possibility 🤣

Reply:
I see now, updated in https://github.com/discord/manifold/pull/13/commits/852d19888e3ee9acf0a88b5ff0a7beb6fe4a6c11